alembic/versions/fc6b0169c596_add_a_column_for_single_table_.py

```diff
@@ -0,0 +1,26 @@
+"""
+Add a column for single-table inheritance to the packages table.
+
+Revision ID: fc6b0169c596
+Revises: 4f2f825bcf4a
+Create Date: 2017-03-31 16:40:07.136469
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'fc6b0169c596'
+down_revision = '4f2f825bcf4a'
+
+
+def upgrade():
+    """Add the column used for polymorphic identity in Packages."""
+    # The default of ``1`` is the RPM package type.
+    op.add_column('packages', sa.Column('type', sa.Integer(), nullable=False, default=1))
```

Comment:
The setting I wanted was ``server_default`` rather than ``default``. This lets you immediately set the column to be non-nullable (because it'll use the default for everything) and then you just alter the column to drop the server default. It comes to the same thing, though.